doc/contrail_deployment/CAD_for_windows_design.md

```diff
@@ -0,0 +1,32 @@
+# Workflow of contrail-ansible-deployer for windows
+
+## Supported orchestrators:
+  * `Openstack` - keystone is needed by windows docker driver
+
+## Playbooks:
+  * `provision_instances.yml` - not supported at this moment
+  * `configure_instances.yml`
+  * `install_openstack.yml`
+  * `install_contrail.yml`
+
+## OS-specific roles:
+  * There are particular roles which needs to be done differently for Windows
+  * Thus we divide these roles into `*_Linux` and `*_Win32NT` for usage in CAD for linux and CAD for windows respectively.
+
+## Configure_instances:
+  * Some differences between windows and non-windows CAD workflow exist for this playbook:
+    * Windows compute nodes need Windows-specific dependencies and particular features turned on/off. `Install_software_Win32.yml` is responsible for that.
+    * certificats?

```

Comment:
Sure. Explain we use self-signed certs for now so we have to enable test-signing (and that because of that we have to reboot each time) and we also have to import them manually